data/json/recipes/electronics/tools.json

```diff
@@ -16,6 +17,7 @@
     "tools": [ [ [ "soldering_iron", 20 ], [ "toolset", 20 ] ] ],
     "components": [
       [ [ "television", 1 ] ],
```

Comment:
The survivor opens the casing and adds some custom circuitry. If it's being produced entirely from spare parts then that's not improvised.